envs/qa.env

```diff
@@ -21,3 +21,4 @@ REWARDING_VALIDATOR_ADDRESS=n1rfvpsynktze6wvn6ldskj8xgwfzzk5v6pnff39
 EXPLORER_API=https://qa-network-explorer.qa.nymte.ch/api
 NYXD=https://qa-validator.qa.nymte.ch
 NYM_API=https://qa-nym-api.qa.nymte.ch/api
+NYM_VPN_API=https://nym-vpn-api-git-deploy-qa-nyx-network-staging.vercel.app/api/
```

Comment:
could you please add the same variable (with appropriate values) to 
- `envs/sandbox.env`
- `envs/canary.env`

Reply:
@dynco-nym bump